srcpkgs/python3-typed-ast/template

```diff
@@ -0,0 +1,19 @@
+# Template file for 'python3-typed-ast'
+pkgname=python3-typed-ast
+version=1.3.1
+revision=1
+archs=noarch
+wrksrc="typed-ast-${version}"
+build_style=python3-module
+hostmakedepends="python3-devel python3-setuptools"
+depends="python3"
```

Comment:
no need to depend on python3, you are compiling `CPython` modules so they call into the python library will be detected and python will be added to the dependencies automatically.

Reply:
in `python3-mypy_extensions` you did exactly the same